src/components/payments-network/payments-network.component.jsx

```diff
@@ -2,11 +2,9 @@ import React from 'react'
 
 import usePaymentsNetworkStyles from './payments-network.styles'
 import Title from '../shared/title/title.component'
-import Button from '../shared/button/button.component'
+// import Button from '../shared/button/button.component'
```

Comment:
Why not remove it?

Reply:
Well it's hidden as the button in the page is temporarily hidden. Maybe I can add a comment for this line as well?